### ISSUE TITLE
Build on DevOps HOSTED Agent

### ISSUE THREAD

**Issue by fernandodsanjos** [BTDF/DeploymentFramework#487]
Is it possible to build BTDF on DevOps HOSTED Agent.

**Comment by tfabraham** [comment]
I don't think so, because at minimum you need to install the Build Tools (I forget the actual name, but it's a feature checkbox) from the BizTalk Server installer to do a BizTalk solution build. I assume you are talking about building a BizTalk app, not BTDF itself?

**Comment by Pauwelz** [comment]
If you have a BizTalk 2020 project you are able to use the Hosted DevOps agents. The 'windows-latest' / 'windows-2019' DevOps agent has the BizTalk Build components deployed. ([Included software for windows-2019](https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md#biztalk-server))

I combine this together with BTDF to create deployments using DevOps Hosted agents.

For this I added the BTDF framework to my DevOps repository and added the following to my Deployment.btdfproj:
`<DeploymentFrameworkRootDir>..\..\..\ALM\BTDF\</DeploymentFrameworkRootDir>
    <DeploymentFrameworkTargetsPath>$(DeploymentFrameworkRootDir)\Framework\</DeploymentFrameworkTargetsPath>`
So it looks something like this:
![image](https://user-images.githubusercontent.com/22931593/150753236-47c508f8-1074-4377-a9a5-6d607018f726.png)

There's an issue with Flat File builds, if you use XSD promotion in Flat Files then the build process will fail. I think that's because the Project Build Component is installed on the Hosted againt, but not the Developer Tools & SDK
